modules/javafx.base/src/main/java/com/sun/javafx/PreviewFeature.java

```diff
@@ -62,7 +74,9 @@ public enum PreviewFeature {
      * @throws RuntimeException if preview features are not enabled
      */
     public void checkEnabled() {
-        if (!enabled) {
+        if (enabledForTesting) {
+            // do nothing
+        } else if (!enabled) {
```

Comment:
`if (!enabledForTesting && !enabled) {` ?

Reply:
I would have to surround both branches (if and else), as otherwise the warning will be emitted during tests. That would, however, increase nesting…